docs/resources/Guild.md

```diff
@@ -521,6 +521,112 @@ A partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object. Represents an Offl
+| ------------------- | --------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------- |
+| guild_id            | snowflake                                                                                                       | ID of the guild this onboarding is part of                 |
+| prompts             | array of [onboarding prompt](#DOCS_RESOURCES_GUILD/guild-onboarding-object-onboarding-prompt-structure) objects | Prompts shown during onboarding and in customize community |
+| default_channel_ids | array of snowflakes                                                                                             | Channel IDs that  members get opted into automatically     |
+| enabled             | boolean                                                                                                         | Whether onboarding is enabled in the guild                 |
+
+
+###### Onboarding Prompt Structure
+
+| Field         | Type                                                                                                    | Description                                                                |
+| ------------- | ------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------- |
+| id            | snowflake                                                                                               | ID of the prompt                                                           |
+| type          | [prompt type](#DOCS_RESOURCES_GUILD/guild-onboarding-object-prompt-type)                                | Type of prompt                                                             |
+| options       | array of [prompt option](#DOCS_RESOURCES_GUILD/guild-onboarding-object-prompt-option-structure) objects | Options available within the prompt                                        |
+| title         | string                                                                                                  | Title of the prompt                                                        |
+| single_select | boolean                                                                                                 | Indicates whether users are limited to selecting one option for the prompt |
+| required      | boolean                                                                                                 | Indicates whether the prompt is required in the onboarding flow             |
+| in_onboarding | boolean                                                                                                 | Indicates whether the prompt is in the onboarding flow                     |
```

Comment:
The description should probably clarify what this meana

Reply:
@advaith1 which one? `in_onboarding`?

Reply:
yeah (I wish gh's ui made it more clear that I selected that line)

Reply:
does the new wording make it a lil clearer?

Reply:
yup! looks good